### ISSUE TITLE
Any update for Yii2??

### ISSUE THREAD

**Issue by frey1esm** [smitherson/humhub-videojs#1]

**Comment by smitherson** [comment]
How soon do you need it?

**Comment by frey1esm** [comment]
Not critical since the included Jplayer can do video and I simply modifiied the showFiles.php view to make it play video.  But eventually I want my website player to match the player on my hybrid app which will be videojs.

Is videojs on a roadmap for humhub?

**Comment by smitherson** [comment]
I'm a freelance developer and I made this plugin for a project.
I won't be doing any HumHub related work any time soon.
But I'll probably take a look at the new Yii framework and if it is easy
i'll make an updated module for videojs.

**Comment by frey1esm** [comment]
Yeah I think it will be simple to convert. There will likely be just a few syntax changes and the need to include external dependencies in the module, I think. 


But I am not Yii/Yii2 expert!

**Comment by smitherson** [comment]
I'll look at it in a few days and write when I have an update. :)

**Comment by frey1esm** [comment]
FYI This helped me: https://github.com/wbraganca/yii2-videojs-widget

**Comment by smitherson** [comment]
Great, so i wont be making any updates, there's no point in having many
modules for one thing. :)

**Comment by frey1esm** [comment]
The one advantage would be the ability to add / remove the module through humhub.
